src/vm/ceeload.cpp

```diff
@@ -10035,9 +10035,9 @@ void Module::RestoreMethodTablePointerRaw(MethodTable ** ppMT,
 
     if (CORCOMPILE_IS_POINTER_TAGGED(fixup))
     {
-#ifdef _WIN64 
+#ifdef _TARGET_64BIT_
```

Comment:
This does not look quite right to me. I think that there is also a problem with `SIZE_T` in `CORCOMPILE_UNTAG_TOKEN` macro. 

Maybe best to postpone this to the "interesting changes" batch.